contracts/plugins/assets/NonFiatCollateral.sol

```diff
@@ -43,7 +43,7 @@ contract NonFiatCollateral is FiatCollateral {
         uint192 pricePerTarget = uoaPerTargetFeed.price(oracleTimeout); // {UoA/target}
 
         // {UoA/tok} = {UoA/target} * {target/ref} * {ref/tok}
```

Comment:
Should clarify comment that this is for when {ref/tok} is 1 which is why it's omitted.

Ideally, we should include refPerTok here to account for plugins that inherit this contract and implement their own refPerTok.